src/SDKs/Automation/Management.Automation/Microsoft.Azure.Management.Automation.csproj

```diff
@@ -1,6 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <!-- Please do not move/edit code below this line -->
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  
+  <PropertyGroup>
+	<PackageId>Microsoft.Azure.Management.Automation</PackageId>
+    <description>Provides Microsoft Azure Automation management operations including the ability to create, update and delete runbooks and schedules.</description>
```

Comment:
Remove lines 5-27
Please include comments above and below line 3 (similar to what you see on line 44, 46)